OWML.ModHelper.Events/OWML.ModHelper.Events.csproj

```diff
@@ -33,35 +33,42 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="0Harmony, Version=1.2.0.1, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
```

Comment:
I had some issues with this reference when I built your branch, I wonder if this is why

Reply:
removed that line